apps/prairielearn/src/tests/groupExam.test.sql

```diff
@@ -0,0 +1,52 @@
+-- BLOCK select_group_work_exam_assessment
+SELECT
+  a.id
+FROM
+  assessments AS a
+  JOIN assessment_sets AS aset ON (aset.id = a.assessment_set_id)
+WHERE
+  a.course_instance_id = 1
+  AND aset.abbreviation = 'E'
+  AND a.group_work is TRUE;
+
+-- BLOCK select_group_config
+SELECT
+  minimum,
+  maximum
+FROM
+  group_configs
+WHERE
+  assessment_id = $assessment_id
+  AND deleted_at IS NULL;
+
+-- BLOCK select_student_user
+SELECT
+  count(*) as result
+FROM
+  users AS u;
```

Comment:
AFAICT this query is unused (and assuming you copied it from `groupStudent.test.sql`, it's unused there as well).